fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,33 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+from subprocess import call, TimeoutExpired, STDOUT
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
+
+
+def main(argv):
+    if len(argv) < 2:
+        raise app.UsageError("Too few arguments")
```

Comment:
I'm not sure the user will know how to use the script if they only get a `Too few arguments` error. Perhaps turn this into something more actionable? Maybe `Missing fuzzer binary path argument.`

Reply:
Changed to
```
This script receives 2 arguments. It should look like:
        python fuzzing/tools/launcher.py EXECUTABLE --timout=TIMEOUT```